React_Day6/avengers/src/utils/apiCall.js

```diff
@@ -0,0 +1,20 @@
+import { userID } from "../constants/constants.js";
+
+async function fetchservice() {
+  const data = [];
+
+  await Promise.all(
+    userID.map((user) => fetch(`https://api.github.com/users/${user}`))
+  )
+    .then((resp) => Promise.all(resp.map((r) => r.json())))
+    .then((result) => {
+      const teamData = result;
+      data.push(teamData);

```

Comment:
instead of push, better to use spreading. data = [...data,...teamData]

Reply:
spread operator!